styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,354 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
+    border: 0;
+    max-height: none;
+    padding: 0;
+  }
+}
+
+.navbar-nav {
```

Comment:
It isnt specific enough unfortunately. Bootstrap has things like .navbar > ul > li > a and navbar-topLink doesnt work outside of the wrapper.